esm-samples/jsapi-node/package.json

```diff
@@ -1,18 +1,20 @@
 {
   "private": true,
   "dependencies": {
-    "@arcgis/core": "^4.18.1",
+    "@arcgis/core": "^4.19.0",
     "abort-controller": "^3.0.0",
-    "cross-fetch": "^3.0.6"
+    "cross-fetch": "^3.1.2",
+    "rollup-plugin-copy": "^3.4.0",
+    "rollup-plugin-delete": "^2.0.0"
   },
   "devDependencies": {
     "@rollup/plugin-commonjs": "^16.0.0",
     "@rollup/plugin-node-resolve": "^10.0.0",
     "rimraf": "^3.0.2",
-    "rollup": "^2.33.1"
+    "rollup": "^2.44.0"
   },
   "scripts": {
-    "build": "npm run clean && rollup -c",
+    "build": "rollup -c",
     "clean": "rimraf dist"
```

Comment:
Update to `rimraf public` or remove this script and `rimraf`.

Reply:
Oops, I missed that one, I'll just remove.